code/addons/docs/template/stories/docspage/description.stories.ts

```diff
@@ -0,0 +1,30 @@
+import globalThis from 'global';
+
+export default {
+  component: globalThis.Components.Button,
+  // FIXME: remove array subcomponents in 7.0?
+  subcomponents: {
+    Pre: globalThis.Components.Pre,
+  },
+  args: { children: 'Click Me!' },
+  parameters: {
+    docs: {
+      description: {
+        component: '**Component** description',
+      },
+    },
```

Comment:
Should we also have examples that set this stuff via `argTypes`?

Reply:
@tmeasday Added a placeholder, although it's not an especially useful test. We should revisit when we do the annotations server.